data/abilities/lateral-movement/65048ec1-f7ca-49d3-9410-10813e472b30.yml

```diff
@@ -11,7 +11,7 @@
         command: |
           $path = "sandcat.go-windows";
           $drive = "\\#{remote.host.fqdn}\C$";
-          Copy-Item -v -Path $path -Destination $drive"\Users\Public\svchost.exe";
+          Copy-Item -v -Path $path -Destination $drive"\Users\Public\s4ndc4t.exe";
         cleanup: |
           $drive = "\\#{remote.host.fqdn}\C$";
           Remove-Item -Path $drive"\Users\Public\svchost.exe" -Force;
```

Comment:
The cleanup command needs to reflect the change to the filename.

Reply:
That's what I get for trying to do this from memory.